Rpackage/R/pruning.R

```diff
@@ -0,0 +1,49 @@
+pruning <- structure(function(tree){
+  ### T1 is the tree after initital pruning of Tmax
+  T1 <- init_pruning(tree)
+  
+  sequential_prune <- function(tree){
+    ### find weakest links
+    wlink <- weakest_link(tree)
+    
+    ### min_gt is same for all weakest links
+    mit_gt <- wlink[1, 1]
+    
+    ### prune the weakest links from tail of tree
+    ### sort solution
+    if(length(order(wlink[, 2]))>1){
+      wlink <- wlink[order(wlink[, 2]),]
+    }
+    print(wlink)
```

Comment:
Don't forget to remove print statement when done.

Reply:
ok